hack/run-cel-validation-test.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(gauravkghildiyal): Make this a "verify" script (so that it runs along
+# with the verification presubmit) once we are confident about these tests.
```

Comment:
@gauravkghildiyal Can you make an issue to track this so we don't lose track of it?

Reply:
Sure I created https://github.com/kubernetes-sigs/gateway-api/issues/2239 to track